tests/foreman/cli/test_medium.py

```diff
@@ -54,6 +54,54 @@ def test_positive_create_1(self, test_data):
         self.assertEqual(new_obj['name'],
                          result.stdout['name'])
 
+    def test_create_medium_with_location(self):
+        """@Test: Check if medium with location can be created
+
+        @Feature: Medium - Positive create
+
+        @Assert: Medium is created and has new location assigned
+
+        """
+        name = gen_string("alpha", 10)
```

Comment:
`name` variable is used just one time, you can drop it and change line 69 to `'name': gen_string("alpha", 10),`